benchcab/data/pbs_jobscript.j2

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#PBS -l wd
+#PBS -l ncpus={{ncpus}}
+#PBS -l mem={{mem}}
+#PBS -l walltime={{walltime}}
+#PBS -q normal
+#PBS -P {{project}}
+#PBS -j oe
+#PBS -m e
+#PBS -l storage={{storage}}
+
+module purge
+{% for module in modules -%}
```

Comment:
Note the -%, which controls whitespace generation in Jinja. 

https://jinja.palletsprojects.com/en/latest/templates/#whitespace-control